pkg/lockfile/fixtures/pnpm/one-package-v6-lockfile.yaml

```diff
@@ -0,0 +1,14 @@
+lockfileVersion: '6.0'
+
+dependencies:
+  acorn:
+    specifier: 8.7.0
+    version: 8.7.0
+
+packages:
+
+  /acorn@8.7.0:
```

Comment:
Can you add a test to make sure we handle packages with "@" in the name properly? 

e.g. https://github.com/privatenumber/minification-benchmarks/blob/c2a2e3e4c2fc3b286a5ad4ef8b3b0d68afa49f68/pnpm-lock.yaml#L208

Reply:
See the resolved conversation below for details on the updated test cases.